opennms-config-model/src/main/resources/xsds/translator-configuration.xsd

```diff
@@ -35,6 +35,7 @@
 				</sequence>
 				<attribute name="type" type="string" use="required" />
 				<attribute name="name" type="string" use="required" />
+				<attribute name="default" type="string" use="optional" />
```

Comment:
Can we change this to:
```XML
<attribute name="default-value" type="string" use="optional" default="Unknown" />
```

Then we don't have to change the XML configuration file.

Reply:
Do we really want all of the assignments to be optional by default now?

Reply:
A default value doesn't fit the logic.

Reply:
I just looked at it closer.  You're right.  It really doesn't.  I guess I really hate the original design of this schema. :)  The "type" attribute is the killer.  #guilty